pkg/beacon/relay/gjkr/protocol.go

```diff
@@ -1655,8 +1668,49 @@ func (rm *CombiningMember) CombineGroupPublicKey() {
 	// Add reconstructed misbehaved members' individual public keys `G * z_m`.
 	for _, peerPublicKey := range rm.reconstructedIndividualPublicKeys {
 		groupPublicKey = new(bn256.G2).Add(groupPublicKey, peerPublicKey)
-
 	}
 
 	rm.groupPublicKey = groupPublicKey
 }
+
+// groupPublicKeyShares combines group public key shares for each
+// individual member in the group. Those group public key shares are
+// needed to perform the verification of relay entry signature shares coming
+// from given group member.
+func (rm *CombiningMember) groupPublicKeyShares() map[group.MemberIndex]*bn256.G2 {
+	groupPublicKeyShares := make(map[group.MemberIndex]*bn256.G2)
+
+	// Calculate group public key shares for all other operating members.
+	for _, receiverID := range rm.group.OperatingMemberIDs() {
+		if receiverID == rm.ID {
+			continue
+		}
+
+		// Calculate first public key share for given receiver based on
+		// current member public key share points.
```

Comment:
```
// Calculate the first public key share for the given receiver based on
// the current member public key share points.
```

Reply:
...or replace `the given receiver` with `the given operating member`, per [this](https://github.com/keep-network/keep-core/pull/1274#discussion_r401466167) comment.